File: Basketball1.py
## Anthony Dike started: 3/2/18
##
## This is VERSION 1 of a program that
## will simulate basketball games like NBA 2k.

import random

'''
Steps:

1) Create team variables.
2) Create max and min score values. (min 98 - max 122)
3) Assign random int to team variables.
4) When program runs, print team scores.

'''

# create max and min scores
maxScore = 122
minScore = 98

# initialize team variables
Team1 = random.randint(minScore, maxScore)
Team2 = random.randint(minScore, maxScore)

# print scores
print("SCORE: Team1 - " + str(Team1) + " Team2 - " + str(Team2))

File: Basketball1.java
// Anthony Dike started: 3/2/18
//
// This is VERSION 1 of a program that
// will simulate basketball games like NBA 2k.

import java.util.Random;

/*
Steps:

1) Initialize team variables and max and min scores
2) Create Random class
3) Assign random int to team variables
4) Convert team int value to string
5) When program runs, print team scores.

*/

public class Basketball1 {
	
	public static void main(String[] arguments){

		// initialize
		int Team1, Team2;
		int max = 122;
		int min = 98;

		// andom class
		Random rand = new Random();

		// assign ints to team variables
		Team1 = rand.nextInt(max - min) + min;
		Team2 = rand.nextInt(max - min) + min;

		// convert int to string
		String team1Value = Integer.toString(Team1);
		String team2Value = Integer.toString(Team2);

		// print scores
		System.out.println();
		System.out.println("SCORE: Team1 - " + team1Value + " Team2 - " + team2Value);
		System.out.println();

	}
}




